docs/cloudfoundry/config_examples/manifest.yaml

```diff
@@ -66,25 +66,40 @@ applications:
 
       # Include the section below to use the cf-bigip-ctlr as a Service Broker
       # THE SETTINGS IN THIS SECTION ARE ROUTE-SPECIFIC
+      # The first health monitor references a monitor that already exists in the cf partition on the BIG-IP device
+      # The second health monitor created a new monitor on the BIG-IP device.
       SERVICE_BROKER_CONFIG: |
-                            plans:
-                              - description: plan for sb test example,
-                                name: sbtest,
-                                virtualServer:
-                                  - policies:
-                                    - /Common/PreventSpoofOfXFF
-                                  - profiles:
-                                    - /Common/x-forwarded-for
-                                  - sslProfiles:
-                                    - /Common/server-ssl
-                                pool:
-                                  balance: ratio-member
-                                  healthMonitors:
-                                    # Use a health monitor that already exists in the cf partition on the BIG-IP device
-                                    - name: /cf/my-healthMonitor
-                                    # Create a new health monitor
-                                    - name: hm-test
-                                      type: http
-                                      interval: 12
-                                      timeout: 5
-                                      send: hello
+                             {
```

Comment:
Just for sanity has this been run through a linter and has it been used in a running CC?

Reply:
I yaml linted it and json linted it with passes. I tried building cf yesterday and it failed so I am in the process of spinning up another cf stack today.